website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -24,9 +24,29 @@
 
 Column width is determined by manually resizing the header column and cells within Figma. As a best practice, column width should be adjusted to fit the longest data type within the cell.
 
-### Alignment
+### Placement
+
+!!! Info
+
+The column placement property is only relevant within Figma and doesn’t exist as a property within the code.
+!!!
+
+Column placement determines the visual styling based on where the column is placed relative to other columns in the Table.
+
+![Table column placement example](/assets/components/table/table-col-placement.png =750x*)
```

Comment:
I am not sure I understand this graphic/visualization. Could it be the same for other readers too?

Reply:
I haven't touched this since we released the component, even though this is a Figma-only property this is a bit ambiguous. Let me see if I can make it a bit more clear